bioseq_tools/dna_rna_fastq_tools.py

```diff
@@ -0,0 +1,64 @@
+from modules.dna_rna_module import transcribe, reverse, complement, reverse_complement
+from modules.fastq_module import count_gc, get_mean_quality, read_fastq, write_fastq
+
+
+def run_dna_rna_tools(*args: str) -> str:
+    """Runs DNA/RNA processing procedures on prescribed sequences"""
+    procedure = args[-1]
+    sequences = args[:-1]
+
+    for seq in sequences:
+        if not set(seq).issubset(set("ATCGUatcgu")) or ("T" in seq and "U" in seq):
+            return f"Incorrect sequence (not DNA or RNA): {seq}"
+
+    if procedure == "transcribe":
+        fn = transcribe
+    elif procedure == "reverse":
+        fn = reverse
+    elif procedure == "complement":
+        fn = complement
+    elif procedure == "reverse_complement":
+        fn = reverse_complement
```

Comment:
```suggestion
operations = {
    'transcribe': utils.transcribe,
    'reverse': utils.reverse,
    'complement': utils.complement,
    'transcribe_dna_complement': utils.transcribe_dna_complement,
    'reverse_complement': utils.reverse_complement
}
```
Можно было бы использовать данный словарь для определения связи между названиями операций и соответствующими функциями.